### PR TITLE
CeleryPurge: log the PHP code that makes the purge

### PR DESCRIPTION
**Pull request by macbre**
Make it a bit easier to debug the purging in MediaWiki:

```
macbre@dev-macbre:~$ tail -f /tmp/debug.log | grep "Purging"

Purging backtrace: MediaWiki::run/MediaWiki::main/MediaWiki::performRequest/MediaWiki::performAction/PurgeAction::show/PurgeAction::onSubmit/Article::doPurge/Article::__call/call_user_func_array/WikiPage::doPurge/SquidUpdate::doUpdate/SquidUpdate::purge/CeleryPurge::purge
Purging URL http://poznan.macbre.wikia-dev.com/wiki/Nowy_Ratusz from PurgeAction:onSubmit via Celery
Purging URL http://poznan.macbre.wikia-dev.com/wiki/Nowy_Ratusz?action=history from PurgeAction:onSubmit via Celery
Purging URL http://poznan.macbre.wikia-dev.com/wikia.php?controller=GameGuides&method=getPage&page=Nowy+Ratusz from PurgeAction:onSubmit via Celery
Purging URL http://poznan.macbre.wikia-dev.com/wikia.php?controller=MercuryApi&method=getArticle&title=Nowy_Ratusz from PurgeAction:onSubmit via Celery
```

This change is a [backport of SquidPurge code](https://github.com/Wikia/app/blob/dev/includes/cache/wikia/ScribePurge.class.php#L43-44)

@owend / @Warkot

### CODE REVIEW

File: includes/cache/wikia/CeleryPurge.class.php [hunk 1]
```diff
@@ -21,11 +21,14 @@ class CeleryPurge {
 
 	// Add some urls to the Task
 	static function purge( $urlArr ) {
+		$caller = self::getPurgeCaller();
 
 		// Filter urls into buckets based on service backend
 		$buckets = array_reduce($urlArr, function($carry, $item) {
 			global $wgPurgeVignetteUsingSurrogateKeys;
 
+			wfDebug( "Purging URL $item from $method via Celery\n" );
```
Comment (owend) [inline includes/cache/wikia/CeleryPurge.class.php]: should be $caller right?

**Comment by Warkot** [comment]
Thanks, @macbre and @owend! :+1:

File: includes/cache/wikia/CeleryPurge.class.php (hunk 1 above)
Comment (macbre) [inline includes/cache/wikia/CeleryPurge.class.php]: Good point, fixed

**Comment by macbre** [comment]
Thanks @owend

**Pull request merged**